### PR TITLE
Export render function

### PR DESCRIPTION
**Pull request by rondale-sc**
Instead of hanging the `render` function off the testContext here we export it as a standalone function.

Imported like so:

```
import { render } from '@glimmer/test-helpers';
```

This mirrors the `@ember/test-helpers` style.

You'll also note that this also introduces a testing harness to the glimmerjs/glimmer-test-helpers repo so we can ensure correctness as we mvoe more of the test harness to this style.

### CODE REVIEW

**Review by rwjblue — APPROVED** [review]
This is great! This will let us actually iterate and test future testing changes "for real" in an app.

Thanks for working on it!

**Comment by rondale-sc** [comment]
Should be good to go.  Travis is happy. 🚦

**Comment by rondale-sc** [comment]
You know, I wonder if this represents a breaking change since `render` was taken off the testContext.

File: src/setup-rendering-test.ts [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-} from "@glimmer/component";
 
+import { setContext } from './setup-context';
 import { getApp } from "./app";
 
-type SerializedTemplateWithLazyBlock = any;
-
 export default function setupRenderingTest(hooks: NestedHooks): void {
   hooks.beforeEach(function beforeEach() {
+    setContext(this);
```
Comment (rwjblue) [inline src/setup-rendering-test.ts:9]: We can mitigate this being a breaking change by adding:

```js
import { render } from './render';
//..snip...
hooks.beforeEach(function beforeEach() {
  setContext(this);
  this.app = new (getApp() as any)();

  this.render = (...args) => {
    console.warn('Using `this.render` in the testing harness is deprecated, please migrate to `import { render } from '@glimmer/test-helpers';` instead.');
    return render(...args);
   };
  //...snip...
});
```

Comment (rondale-sc) [reply src/setup-rendering-test.ts:9]: This seems like a good idea.  Will implement now

**Comment by rondale-sc** [comment]
@rwjblue 	Should be gtg now.  Added a test to ensure it worked.  Turns out having a test harness is fun!  💯

**Pull request merged**